modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc

```diff
@@ -13,7 +13,7 @@ When you create a cluster, you select a xref:deploy:deployment-option/cloud/clou
 
 === AWS PrivateLink
 
-xref:deploy:deployment-option/cloud/aws-privatelink.adoc[AWS PrivateLink] is now available as an easy and highly secure way to connect to Redpanda Cloud from your VPC. You can set up the PrivateLink endpoint service for a new cluster or an existing cluster. To enable AWS PrivateLink for you account, contact https://support.redpanda.com/hc/en-us/requests/new[Redpanda support].
+xref:deploy:deployment-option/cloud/aws-privatelink.adoc[AWS PrivateLink] is now available as an easy and highly secure way to connect to Redpanda Cloud from your VPC. You can set up the PrivateLink endpoint service for a new cluster or an existing cluster. To enable AWS PrivateLink for your account, contact https://support.redpanda.com/hc/en-us/requests/new[Redpanda support].
```

Comment:
```suggestion
xref:deploy:deployment-option/cloud/aws-privatelink.adoc[AWS PrivateLink] is now available as an easy and highly secure way to connect to Redpanda Cloud from your VPC. You can set up the PrivateLink endpoint service for a new cluster or an existing cluster. To enable AWS PrivateLink for your account, contact https://support.redpanda.com/hc/en-us/requests/new[Redpanda support^].
```
@kbatuigas adding the ^ to make the external link open in a new tab + would you please move this up above Additional cloud tiers, so it's top down with most recent addition?